server/server/src/lib.rs

```diff
@@ -261,8 +261,6 @@ pub async fn start_server(
     BoaJsContext::new(
         &service_provider,
         PluginExecuteGraphql(graphql_schema.clone()),
-        // See explanation in BoaJsContext::new and in do_async_blocking
-        tokio::runtime::Handle::current(),
```

Comment:
This is not needed anymore, we are running async in sync method with a new thread, as per PR description